robottelo/virtwho_utils.py

```diff
@@ -0,0 +1,400 @@
+"""Utility module to handle the virtwho configure UI/CLI/API testing"""
```

Comment:
Would it make sense to merge this entire file with `robottelo/helpers.py`?

`helpers.py` is a little mess, but I would rather have one helpers file than one for virtwho and one for everything else.

Reply:
I saw `helpers.py` already has many functions, in my opinion, put these virt-who related functions to a file would be better. What is your opinion? @eko999 As Eko designed this file.  :)

Reply:
I prefer to keep virtwho utility file independent, the functions only can be used for virt-who module, they are not public functions for other modules. If these two files are merged together, helpers.py will be longer, and it's not so easy to know what the function is designed for, unless we rename the function name.